github_rules/github_repo_initial_access.yml

```diff
@@ -0,0 +1,78 @@
+AnalysisType: rule
+Filename: github_repo_initial_access.py
+RuleID: GitHub.Repo.InitialAccess
+DisplayName: GitHub Branch Protection Policy Override
+Enabled: false
+LogTypes:
+  - GitHub.Audit
+Tags:
+  - GitHub
+Severity: Info
```

Comment:
I might even call this a `Low` severity event, but I do see the case for `Info`. Definitely a tricky one.

Reply:
yea... I was torn.  I was thinking with really large orgs this might be noisy, but I still think its valuable to see this (vs auto-resolved INFO) if for nothing else that to notify folks that a user is accessing a repo maybe they shouldn't have access to (fix bad perms)